### DISCUSSION TITLE
LAMMPS error: libgomp: Thread creation failed: Resource temporarily unavailable

### DISCUSSION THREAD

**Discussion by singlebook** [deepmodeling/deepmd-kit#315] [Q&A] [answered]
Hello DeepMD users,

I run CPU based LAMMPS with trained deepmd potential, I always got an error :

libgomp: Thread creation failed: Resource temporarily unavailable

It seems that Tensorflow is consuming too much memory. 

What should I do to reduce the memory cost?

Your response is highly appreciated.
Best wishes,
Wei

**Comment by njzjz** [comment, marked as answer]
https://stackoverflow.com/a/53379624/9567349 Is this answer helpful?

**Reply by singlebook** [reply]
Dear Dr. Zeng,

Thanks very much for your reply.

export OMP_NUM_THREADS=1
export TF_INTRA_OP_PARALLELISM_THREADS=1
export TF_INTER_OP_PARALLELISM_THREADS=1

These three commands save me. But I get new warnings. 

WARNING: Radial neighbor list length of type 0 is not enough
WARNING: Radial neighbor list length of type 1 is not enough

is the value of sel not large enough?

Best wishes,
Wei

**Reply by njzjz** [reply]
Yes. See #108.
